testing/zip_iterator.cu

```diff
@@ -276,13 +276,14 @@ template <typename Vector>
 void TestZipIteratorCopy(void)
 {
   using namespace thrust;
+  using T = typename Vector::value_type;
 
   Vector input0(4),  input1(4);
   Vector output0(4), output1(4);
 
   // initialize input
-  sequence(input0.begin(), input0.end(),  0);
-  sequence(input1.begin(), input1.end(), 13);
+  sequence(input0.begin(), input0.end(),  T{0});
```

Comment:
The extra space is a bit odd now that we've added the temporary.

Reply:
Fixed.